lib/posix/options/signal.c

```diff
@@ -13,8 +13,7 @@
 #define SIGNO_WORD_IDX(_signo) (signo / BITS_PER_LONG)
 #define SIGNO_WORD_BIT(_signo) (signo & BIT_MASK(LOG2(BITS_PER_LONG)))
 
-BUILD_ASSERT(CONFIG_POSIX_LIMITS_RTSIG_MAX >= 0);
-BUILD_ASSERT(CONFIG_POSIX_RTSIG_MAX >= CONFIG_POSIX_LIMITS_RTSIG_MAX);
+BUILD_ASSERT(CONFIG_POSIX_RTSIG_MAX >= CONFIG_POSIX_RTSIG_MAX);
```

Comment:
Should this be `>= 8`?

Reply:
It is 8 by default.

Reply:
Ah.. it doesn't always have to be 8, probably this should be changed to
```suggestion
BUILD_ASSERT(CONFIG_POSIX_RTSIG_MAX >= 0);
```

otherwise it would be comparing against itself and is always true